### PR TITLE
Fix French flag.

### PR DESCRIPTION
**Pull request by benguillet**
Fix the french flag :) (blue <-> red).

The diff is bigger than necessary because my editor automatically strips all the trailing whitespaces, which I think you should do too, so that in future pull requests no random whitespaces appear.

### CODE REVIEW

**Comment by cmaddux** [comment]
I will definitely work on stripping the whitespace, good call. Thanks for the heads up and for fixing up the French flag!

File: lib/littlebox.css [hunk 1]
```diff
@@ -3261,26 +3261,26 @@
 .lb-flag-france {
```
Comment (geoffrey) [inline lib/littlebox.css:3261]: change the above comment as well :p

**Comment by benguillet** [comment]
Hey thanks for the sweet library :)

File: lib/littlebox.css (hunk 1 above)
Comment (cmaddux) [inline lib/littlebox.css:3261]: Thanks for catching that! There were a few leftovers in there. Getting rid of them now.

Comment (geoffrey) [inline lib/littlebox.css:3261]: Hehe was jk, it's a very neat library, keep up the good work :beers:

Comment (cmaddux) [inline lib/littlebox.css:3261]: Probably still good to get them out of there though ha. They did not look good. Thanks for checking it out!

Comment (cmaddux) [inline lib/littlebox.css:3261]: Aaah. Just found the misplace flag-canada note ha sorry about that.  I may never be allowed back into Paris. Thanks!

**Pull request merged**